src/applications/personalization/preferences/helpers.jsx

```diff
@@ -1,56 +1,234 @@
+import React from 'react';
+
+const healthFAQ = () => (
+  <div>
+    <ul>
+      <li>
+        <a href="/health-care/eligibility/">
+          See if you’re eligible for VA health care benefits.
+        </a>
+      </li>
+      <li>
+        <a href="/health-care/how-to-apply/">
+          Learn how to apply online, by mail, or in person.
+        </a>
+      </li>
+      <li>
+        <a href="/health-care/about-va-health-benefits/">
+          Find out what kinds of health care and services are covered.
+        </a>
+      </li>
+    </ul>
+  </div>
+);
+
+const disabilityFAQ = () => (
+  <div>
+    <ul>
+      <li>
+        <a href="/disability/eligibility">
+          See if you’re eligible for compensation.
+        </a>
+      </li>
+      <li>
+        <a href="/disability/how-to-file-claim/when-to-file/">
+          Learn about the different claim types.
+        </a>
+      </li>
+      <li>
+        <a href="/disability/get-help-filing-claim">
+          Consider working with a trained professional who can help you file
+          your claim.
+        </a>
+      </li>
+      <li>
+        <a href="/disability/how-to-file-claim">
+          Find out how to file a claim online, by mail, or in person.
+        </a>
+      </li>
+      <li>
+        <a href="/disability/file-an-appeal">
+          Learn how to file an appeal if you disagree with our decision on your
+          claim.
+        </a>
+      </li>
+    </ul>
+    <h5>You may also be interested in:</h5>
+    <ul>
+      <li>
+        <a href="/careers-employment/vocational-rehabilitation/">
+          Vocational Rehabilitation and Employment
+        </a>
+      </li>
+      <li>
+        <a href="/housing-assistance/disability-housing-grants/">
+          Adaptive Housing Grants
+        </a>
+      </li>
+      <li>
+        <a href="/pension/">VA pension benefits</a>
+      </li>
+      <li>
+        <a href="/health-care/family-caregiver-benefits/comprehensive-assistance/">
+          The Program of Comprehensive Assistance to Family Caregivers of
+          Post-9/11 Veterans
+        </a>
+      </li>
+    </ul>
+  </div>
+);
+
+const educationFAQ = () => (
+  <div>
```

Comment:
we probably don't need the wrapper `div`

Reply:
Good call.